include/cpuinfo.h

```diff
@@ -34,6 +34,16 @@
 	#define CPUINFO_ARCH_PPC64 1
 #endif
 
+#if defined(__riscv)
+    #if __riscv_xlen == 32
+        #define CPUINFO_ARCH_RISCV 1
```

Comment:
Should this define be `CPUINFO_ARCH_RISCV32`? Unlike X86 and ARM, RISCV32 and RISCV64 were developed at the same time so 64 isn't an addition.